packages/esm-patient-list-app/src/patient-list-detail/patient-list-detail.test.tsx

```diff
@@ -82,19 +82,31 @@ describe('PatientListDetailComponent', () => {
     });
   });
 
-  it('opens edit overlay when "Edit Name/ Description" is clicked', () => {
+  it('opens edit overlay when "Edit Name or Description" is clicked', () => {
     render(<PatientListDetailComponent />);
 
     userEvent.click(screen.getByText('Actions'));
-    const editBtn = screen.getByText('Edit Name/ Description');
+    const editBtn = screen.getByText('Edit Name or Description');
     userEvent.click(editBtn);
   });
 
   it('deletes patient list and navigates on successful delete', async () => {
     render(<PatientListDetailComponent />);
 
     await waitFor(() => {
-      userEvent.click(screen.getByText('Delete'));
+      userEvent.click(screen.getByText('Actions'));
+      userEvent.click(screen.getByText('Delete patient List'));
+    });
+
+    await waitFor(() => {
+      expect(screen.getByText('Are you sure you want to delete the patient list')).toBeInTheDocument();
```

Comment:
Note that this string will not match `Are you sure you want to delete {patientListName} patient list?`.

Reply:
Just a thought - a possible variation of this translation that’s more flexible is: Are you sure you want to delete the list named {listName}

Reply:
@ibacher, @denniskigen and @hadijahkyampeire thanks for the review and guidance.